app/build.gradle.kts

```diff
@@ -34,7 +34,7 @@ android {
 
     defaultConfig {
         applicationId = "com.example.platform.app"
-        minSdk = 21
```

Comment:
We're not looking to upgrade our minSdk to 24. You should use `RequiresApi` annotation around your sample instead